src/skills-builder/skills-builder-modal/SkillsBuilderModal.jsx

```diff
@@ -28,14 +29,42 @@ const SkillsBuilderModal = () => {
     history.goBack();
   };
 
+  const sendActionButtonEvent = (eventSuffix) => {
+    sendTrackEvent(
+      `edx.skills_builder.${eventSuffix}`,
+      {
+        app_name: 'skills_builder',
```

Comment:
Sorry to re-hash this. Should the "app_name" here be "profile"? Since it's coming from the Profile MFE? This might be able to help us differentiate metrics from this features life as part of the profile MFE (compared to the dedicated MFE we expect to build for it). Am I just thinking about this wrong? I know we talked about this during standup the other day but I think I'm still a little fuzzy here